oidc-client-app-samples/browser-client-app-sample/SPA_HTML_app/js/app.js

```diff
@@ -25,27 +25,64 @@
-  'flowType': "PKCE", // Possible values are IMPLICIT and PKCE, default is IMPLICIT
-  'userStore': "LOCAL_STORAGE", // Possible values are LOCAL_STORAGE and SESSION_STORAGE, default is LOCAL_STORAGE and SESSION_STORAGE is not not supported yet
-  'clientId': '0Ob7gGlPWWoCMx4qrGNSWjRgnMwa', // Get this value from WSO2-IAM Server Service Provider (SP) definition
-  'clientSecret': '4jjZU5nke4iovlq5seaSPVxnqBMa', // Get this value from WSO2-IAM Server Service Provider (SP) definition
-  'redirectUri': 'http://localhost:8080/app/', // Specify the redirect URL to land back after login redirect to WSO2-IAM Server login page
-  'scope': 'openid', // This is to specify that you are using openID Connect
-  'postLogoutRedirectUri': 'http://localhost:8080/app/' // Specify the post logout redirect URL to land back after logout redirect to WSO2-IAM Server logout page
+  // Possible values are FLOW_TYPE_IMPLICIT and FLOW_TYPE_IMPLICIT, default is FLOW_TYPE_IMPLICIT
+  'flowType': FLOW_TYPE_IMPLICIT,
+  // Possible values are LOCAL_STORAGE and SESSION_STORAGE, default is LOCAL_STORAGE and SESSION_STORAGE is not not supported yet
+  'userStore': LOCAL_STORAGE,
+  // Get this value from WSO2-IAM Server Service Provider (SP) definition
+  'clientId': 'YDMv8lTAWH42jYiZkdvqHTocFXsa',
+  // Get this value from WSO2-IAM Server Service Provider (SP) definition
+  'clientSecret': 'dLKJZIvPZX5H_GhTItlQ1eTNEE8a',
+  // Specify the redirect URL to land back after login redirect to WSO2-IAM Server login page
+  'redirectUri': 'http://localhost:8080/SPA_HTML_app/',
+  // This is to specify that you are using openID Connect
+  'scope': 'openid',
+  // Specify the post logout redirect URL to land back after logout redirect to WSO2-IAM Server logout page
+  'postLogoutRedirectUri': 'http://localhost:8080/SPA_HTML_app/'
 };
 
+var app = null;
+var appLogout = null;
+var appPKCE = null;
+var appUserInfo = null;
+
 var init = function (authCompletionCallback, logoutCompletionCallback) {
-  var application = new App(appConfigs);
 
-  application.init(authCompletionCallback, logoutCompletionCallback);
+  // Remove any object initialization of you are not using and remove the respective js library file in js folder.
+  app = new App(appConfigs);
+  appLogout = new AppLogout(app, appConfigs.postLogoutRedirectUri, appConfigs.clientId);
+  appPKCE = new AppPKCE(app, appConfigs.clientId, appConfigs.clientSecret, appConfigs.redirectUri, 
+    appConfigs.scope, appConfigs.tokenUrl);
+  appUserInfo = new AppUserInfo(app, appConfigs.userInfoUrl);
+
+  var appArray = [app, appLogout, appPKCE, appUserInfo];
+
+  app.init(authCompletionCallback);
+  appLogout.init(logoutCompletionCallback);
+  appPKCE.init(authCompletionCallback);
+  // There is no init function in AppUserInfo class
 
   // check for authorization response if available.
-  application.checkForAuthorizationResponse();
+  if (appConfigs.flowType == FLOW_TYPE_IMPLICIT) {
```

Comment:
bring the static value to the front,

FLOW_TYPE_IMPLICIT == appConfigs.flowType

Reply:
done